pkg/lib/notifications.js

```diff
@@ -55,7 +55,10 @@ fields:
  An arbitrary value.  The "System" overview page might monitor a
  couple of pages for their status and it will use 'details' to display
  a richer version of the status than possible with just type and
- title.
+ title. The recognized properties are:
+
+   * icon: custom icon name (defaults to standard icon corresponding to type)
+   * link: custom link target (defaults to page name)
```

Comment:
What about making it so that `link: false` means "not a link"?

Reply:
Sounds good! Done.